crates/sui-core/src/quorum_driver/mod.rs

```diff
@@ -39,7 +41,7 @@ use self::reconfig_observer::ReconfigObserver;
 mod tests;
 
 const TASK_QUEUE_SIZE: usize = 10000;
-const EFFECTS_QUEUE_SIZE: usize = 1000;
+const EFFECTS_QUEUE_SIZE: usize = 10000;
```

Comment:
Is the queue size monitored?

Reply:
it is not. Are we talking about `metered_channel` or another primitive ?

Reply:
Yeah, not asking for it in this PR. But I assume we want to know about backlogs in this queue eventually. Something similar to `metered_channel` (but uses label per channel instead of metric per channel) may be useful.

Reply:
created https://github.com/MystenLabs/sui/issues/7672